src/components/FlowRunTimelineOptions.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <p-pop-over
+    ref="popOver"
+    class="flow-run-timeline-options"
+    auto-close
+    :placement="placement"
+  >
+    <template #target="{ toggle }">
+      <p-button
+        ref="settingsButton"
+        aria-label="Flow Run Timeline Options"
+        icon="CogIcon"
+        flat
+        @click="toggle"
+      />
+    </template>
+    <div class="flow-run-timeline-options__popover">
+      <p-overflow-menu>
+        <div class="flow-run-timeline-options__popover-option-set">
```

Comment:
Is there a reason for using this div and custom styling rather than using the `p-overflow-menu-item` component's with its slot?

Reply:
So `p-overflow-menu-item` was wrapping everything in a button. I don't believe having a radio group in a button is valid markup. Also it caused the whole area to have a `cursor: pointer` when I'd prefer it to only apply to the individual radio/checkbox items themselves.

Reply:
You're correct. Completely forgot that component was a button. Seems like a problem we should solve in prefect-design. I'll open an issue.